### PR TITLE
trying to fix bootstrap code

### PR DESCRIPTION
**Pull request by chensong1995**
* Swift compiler is now built with debugging symbols
* WALA build script fails to fetch dx.jar (from android sdk). This problem is temporarily fixed in the bootstrap code.
* CMakeLists.txt requires several environment variables. They are set in the bootstrap code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CMakeList.txt
```
Comment (Leeleo3x) [inline .gitignore]: This is the configuration file for CMake and should not be ignored.

File: utils/bootstrap.py [hunk 2]
```diff
@@ -37,13 +56,45 @@ def download_and_build_swift_wala(dir_path):
     subprocess.check_call(['ant'], cwd=os.path.join(dir_path, 'swift-wala',
         'com.ibm.wala.cast.swift'))
 
+def set_variables_in_cmake(dir_path):
```
Comment (Leeleo3x) [inline utils/bootstrap.py]: Use -DVARIABLE=VALUE to pass the CMake configurations.

**Comment by karimhamdanali** [comment]
Please tag me in a response when both issues have been resolved.

File: utils/bootstrap.py (hunk 2 above)
Comment (karimhamdanali) [reply utils/bootstrap.py]: Is that now fixed?

File: .gitignore (hunk 1 above)
Comment (karimhamdanali) [reply .gitignore]: Is that now fixed?

**Comment by Leeleo3x** [comment]
Ah, since I have moved the code to swift-wala, we don't need this repo
anymore.

**Comment by karimhamdanali** [comment]
I see. In that case, I will close this PR, archive the repo, and make it private.

**Pull request closed without merging**